pkg/controller/certificates/signer/cfssl_signer.go

```diff
@@ -47,6 +47,7 @@ func NewCSRSigningController(
 		return nil, err
 	}
 	return certificates.NewCertificateController(
+		"signer",
```

Comment:
should this match the name used in the controller manager (e.g. `csrsigning`)?

Reply:
> should this match the name used in the controller manager (e.g. `csrsigning`)?

That would cause `certificate_controller.go:114] Starting certificate controller "csrsigning"` instead of `certificate_controller.go:114] Starting certificate controller "signer"`.  I have no real preference if you do though.